ros_ign_image/package.xml

```diff
@@ -7,8 +7,8 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <depend>ignition-msgs4</depend>
-  <depend>ignition-transport7</depend>
```

Comment:
`ros_ign_bridge` also needs a `package.xml` update

Reply:
I think that we have now them updated after the merge of https://github.com/osrf/ros_ign/pull/58